.github/workflows/build.yml

```diff
@@ -0,0 +1,20 @@
+name: Build + Test
+on: [push]
+
+jobs:
+  build:
+    name: Build + Test
+    runs-on: ubuntu-latest
+
+    steps:
+    - uses: actions/checkout@master
+    - name: Set up Ruby 2.6
+      uses: actions/setup-ruby@v1
+      with:
+        version: 2.6.x
+    - name: Build and test with Rake
+      run: |
+        gem uninstall bundler || true
+        gem install bundler --version '1.0.0'
```

Comment:
Do we need to lock it to this ancient version?

Reply:
Hmmm I was encountering errors with a newer version but let me try again and see what happens.

Reply:
https://github.com/anglinb/hackerone-client/commit/13652d2fa9b08803a6aed5b2e30b429b32da96bd/checks?check_suite_id=284975824#step:4:23

```
Bundler could not find compatible versions for gem "bundler":
  In Gemfile:
    bundler (~> 1.14)

  Current Bundler version:
    bundler (2.0.2)
This Gemfile requires a different version of Bundler.
Perhaps you need to update Bundler by running `gem install bundler`?

Could not find gem 'bundler (~> 1.14)' in any of the relevant sources:
  the local ruby installation
##[error]Process completed with exit code 6.
```

Reply:
Whoops, I merged because it was green 😆

Reply:
Lol you can change it to this and it will work: https://github.com/anglinb/hackerone-client/commit/d3200160e1fdac814202fe7535c419b3e0941418

Reply:
Happy to open another PR or just let you push to master hehe